### DISCUSSION TITLE
Neither map nor cmap show anything and same for calcdirsize

### DISCUSSION THREAD

**Discussion by Lassebq** [gokcehan/lf#1220] [Q&A] [answered]
What am I doing wrong and where should it be displayed?

**Comment by joelim-work** [comment, marked as answer]
`map` and `cmap` just assigns keybindings without printing anything. If you want to view the list of keybindings then use `maps` and `cmaps` (with an 's' on the end), although this is only available starting from version r29.

---

`calcdirsize` is a command that calculates the total size of a directory, but it is only visible if you set `lf` to display file sizes. To see the effect of this, do the following:

1. Set `lf` to display file sizes, by either typing `:set info size` or pressing <kbd>zs</kbd>.
2. Files should now display their size, but for directories `-` is displayed.
3. Select a directory (you can also select multiple directories by pressing <kbd>Space</kbd>).
4. Type `:calcdirsize`, and the size of the selected directories should update.
